packages/application-shell/src/components/navbar/use-navbar-state-manager.ts

```diff
@@ -129,9 +138,10 @@ const useNavbarStateManager = (props: HookProps) => {
     ? null
     : (JSON.parse(cachedIsForcedMenuOpen) as boolean);
 
-  const [state, dispatch] = React.useReducer<
-    (prevState: State, action: Action) => State
```

Comment:
this is a bit too verbose IMO, so I decided to remove it.
It is already inferred through the reducer signature 😉

Reply:
Nice if it is inferred than we can remove it surely.